tests/suites/test_suite_entropy.function

```diff
@@ -310,7 +311,7 @@ void entropy_nv_seed( char *read_seed_str )
     unsigned char entropy[MBEDTLS_ENTROPY_BLOCK_SIZE];
     unsigned char buf[MBEDTLS_ENTROPY_BLOCK_SIZE];
     unsigned char empty[MBEDTLS_ENTROPY_BLOCK_SIZE];
-    unsigned char read_seed[MBEDTLS_ENTROPY_BLOCK_SIZE];
+    unsigned char read_seed->x[MBEDTLS_ENTROPY_BLOCK_SIZE];
```

Comment:
This breaks compilation, not compiled with default configuration but once the required defs are enabled this breaks.